components/mintKamon/GenerateImageForm.tsx

```diff
@@ -1,13 +1,126 @@
-import { Button, Heading } from "@chakra-ui/react"
-import { Layout } from "@/components/layouts/layout"
+import { useAccount } from 'wagmi'
+import { useEffect, useState } from 'react'
+import { Button, Heading, Text, FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { Layout } from '@/components/layouts/layout'
 
-const GenerateImageForm = () => {
+interface Token {
+  name: string,
+  description: string,
+  image: string,
+  attributes: any[],
+}
+
+interface Prop {
+  onSetTokenURI: (value: string) => void
+}
+
+const GenerateImageForm: React.FC<Prop> = ({onSetTokenURI}) => {
+  const ipfsApiEndpoint = 'https://api.staging.sakazuki.xyz/henkaku/ipfs'
+  const { data } = useAccount()
+  const [input, setInput] = useState('')
+  const [disabled, setDisabled] = useState(false)
+  const [tokenJson, setTokenJson] = useState<Token>()
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => setInput(e.target.value)
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => setInput(e.target.files[0])
```

Comment:
```suggestion
  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) =>
    setInput(
      e.target.files && e.target.files.length > 0
        ? e.target.files[0].toString()
        : ''
    )
```

A little aggressive, but at first, fix the build error.